cognite/neat/_store/_base.py

```diff
@@ -244,33 +244,36 @@ def read(self, class_: str) -> Iterable[tuple[str, dict[str | InstanceType, list
     def _parse_file(
         self,
         filepath: Path,
-        mime_type: str = "application/rdf+xml",
+        format: str = "turtle",
         base_uri: URIRef | None = None,
     ) -> None:
         """Imports graph data from file.
 
         Args:
             filepath : File path to file containing graph data, by default None
-            mime_type : MIME type of graph data, by default "application/rdf+xml"
-            base_uri : Add base IRI to graph, by default True
+            format : rdflib format file containing RDF graph, by default "turtle"
+            base_uri : base URI to add to graph in case of relative URIs, by default None
+
+        !!! note "Oxigraph store"
+            By default we are using non-transactional mode for parsing RDF files.
+            This gives us a significant performance boost when importing large RDF files.
+            Underhood of rdflib we are triggering oxrdflib plugin which in respect
+            calls `bulk_load` method from oxigraph store. See more at:
+            https://pyoxigraph.readthedocs.io/en/stable/store.html#pyoxigraph.Store.bulk_load
         """
 
         # Oxigraph store, do not want to type hint this as it is an optional dependency
         if type(self.graph.store).__name__ == "OxigraphStore":
-
```

Comment:
low-level code of oxistore is not anymore needed after update to latest oxrdflib and pyoxi